.circleci/config.yml

```diff
@@ -69,6 +69,12 @@ aliases:
       branches:
         only: /^release\/.*\.0$/
 
+  # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
```

Comment:
I'm not currently using this because the other filter covers it too so it'd be redundant, but this feels like knowledge that is worth preserving in case it's useful in the future.

Reply:
Hmm I thought we weren't running tests on forks... If we are, we should be careful with the usage.

Reply:
removing